jib-core/src/main/java/com/google/cloud/tools/jib/configuration/LayerConfiguration.java

```diff
@@ -45,20 +47,47 @@ public Builder setLabel(String label) {
     }
 
     /**
-     * Adds an entry to the layer.
+     * Adds an entry to the layer. Only adds the single source file to the exact path in the
```

Comment:
The _file_ in `sourceFile` somewhat implies it shouldn't be a directory. _sourcePath_?  And add a note that the contents of a directory are not added.

Reply:
I've been using file to mean directories and files in this codebase. Do you think we should have file only mean regular files for the Jib Core API?

Reply:
Did you mean `LayerEntry::new`? Yes that should be explaining the same thing, although `LayerEntry` is intended to be an internal class (no user-facing API accepts a `LayerEntry`) whereas `LayerConfiguration` is a user-facing API. I'll change the javadocs to be more aligned.